src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
     <p>In the next section we use Extreme Value concepts to <em>optimize</em> quantities.</p>
   </introduction>
   <exercises>
+<!--Group 1: T/F questions-->
     <exercisegroup>
       <introduction>
         <p>Terms and Concepts</p>
       </introduction>
+<!--Exercise 1: T/F-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$true">
+              <set>
+                <member correct="yes">True</member>
+                <member>False</member>
+              </set>
+            </var>
+            <var name="$false">
+              <set>
+                <member>True</member>
+                <member correct="yes">False</member>
+              </set>
             </var>
             <pg-code>
+              $true=PopUp(['?','True','False'],1);
+              $false=PopUp(['?','True','False'],2);
             </pg-code>
           </setup>
           <statement>
-            <p>T/F: Implicit differentiation is often used when solving <q>related rates</q> type problems.</p>
+            <p>T/F: Implicit differentiation is often used when solving <q>related rates</q> type problems. <var name="$true" form="popup"/></p>
```

Comment:
The print versions will looks something like:

Implicit differentiation is often used when solving "related rates" type problems. [Choose: True | False]

Styling may change on that bit at the end, but every place there is a popup answer in the web version, there will be a printed list in the paper version.